cmd/release/cmd/tag.go

```diff
@@ -204,13 +206,71 @@ var systemAgentInstallerK3sTagSubCmd = &cobra.Command{
 	},
 }
 
+var uiTagSubCmd = &cobra.Command{
+	Use:   "ui [ga,rc] [version]",
+	Short: "Tag ui releases",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 2 {
+			return errors.New("expected at least two arguments: [ga,rc] [version]")
+		}
+		rc, err := releaseTypePreRelease(args[0])
+		if err != nil {
+			return err
+		}
+		tag := args[1]
+		uiRelease, found := rootConfig.UI.Versions[tag]
+		if !found {
+			return errors.New("verify your config file, version not found: " + tag)
+		}
+		ctx := context.Background()
+		ghClient := repository.NewGithub(ctx, rootConfig.Auth.GithubToken)
+		opts := &repository.CreateReleaseOpts{
+			Tag:    tag,
+			Repo:   "ui",
+			Owner:  uiRelease.UIRepoOwner,
+			Branch: uiRelease.ReleaseBranch,
+		}
+		return ui.CreateRelease(ctx, ghClient, &uiRelease, opts, rc)
+	},
+}
+
+var dashboardTagSubCmd = &cobra.Command{
+	Use:   "dashboard [ga,rc] [version]",
+	Short: "Tag dashboard releases",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 2 {
```

Comment:
Same thing here, see about doing validations on the standard Validation Arguments function of cobra like:
https://github.com/rancher/ecm-distro-tools/blob/master/cmd/release/cmd/push.go#L47